src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -12,7 +12,7 @@
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.Remark;
-import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.*;
```

Comment:
Ditto here on imports.

Reply:
If i'm not wrong, this has already been corrected.